Resources/Prototypes/_Scp/Entities/Mobs/NPCs/silicon.yml

```
@@ -7,13 +7,22 @@
     blackboard:
       MeleeRange: !type:Single
-        0.1
\ No newline at end of file
+        0.1
+  - type: AmbientSound
+    enabled: true
+    volume: -7
+    range: 5
+    sound:
+      path: /Audio/_Sunrise/Effects/Flying/medibot.ogg
+  - type: FootstepModifier # Из-за какого-то бага не работает
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)